README.md

```diff
@@ -16,6 +16,9 @@ Requirements
 * python
```

Comment:
I think adding version information here would be better.
Like `python (version 2.x)`